backend/src/test/java/com/woowacourse/kkogkkog/application/MemberServiceTest.java

```diff
@@ -76,13 +81,13 @@ void success() {
             SlackUserInfo slackUserInfo = new SlackUserInfo("URookie", "T03LX3C5540", "루키",
                 "rookie@gmail.com", "image", "workspace_name");
             Long memberId = memberService.saveOrFind(slackUserInfo).getId();
+            workspaceRepository.save(new Workspace(null, "T03LX3C5540", "꼭꼭", null));
 
             MyProfileResponse memberResponse = memberService.findById(memberId);
 
             assertThat(memberResponse).usingRecursiveComparison().ignoringFields("id").isEqualTo(
-                new MyProfileResponse(null, "URookie", "T03LX3C5540", "루키", "rookie@gmail.com",
-                    "image", 0L)
-            );
+                new MyProfileResponse(null, "URookie", "T03LX3C5540", "꼭꼭", "루키",
+                    "rookie@gmail.com", "image", 0L));
```

Comment:
"꼭꼭"보다는 좀 더 워크스페이스스러운 네이밍이면 이해하기 편할 것 같은데 어떤가요?

Reply:
```suggestion
                new MyProfileResponse(null, "URookie", "T03LX3C5540", "workspace_name", "루키",
                    "rookie@gmail.com", "image", 0L));
```
[workspace_name을 실제 팀명인 꼭꼭으로 변경](https://github.com/woowacourse-teams/2022-kkogkkog/pull/196/commits/360fc94f1faa4a4cf8add55794b2378480f24524) 이전으로 돌리겠습니다. 더 좋은 이름이 있으면 말씀해주세요.